docs/source/api/apollo-gateway.mdx

```diff
@@ -124,7 +124,7 @@ A [supergraph schema](../#federated-schemas) ([generated with the Rover CLI](htt
 </td>
 <td>
 
-**This option is discouraged in favor of [`supergraphSdl`](#supergraphsdl).**
+**This option is deprecated in favor of the drop-in replacement, [`IntrospectAndCompose`](#class-introspectandcompose).**
```

Comment:
Also don't forget to actually document `supergraphSdl` including the various options it takes like healthCheck and getDataSource.

Reply:
c0b3c40